examples/rbac-remote/README.md

```diff
@@ -0,0 +1,171 @@
+# Feast Deployment with RBAC
+
+## Demo Summary
+This demo showcases how to enable Role-Based Access Control (RBAC) for Feast using Kubernetes or OIDC Authentication type. 
+The demo steps involve deploying server components (registry, offline, online) and client examples within a Kubernetes environment. 
+The goal is to ensure secure access control based on user roles and permissions. For understanding the Feast RBAC framework 
+Please read these reference documents.
+- [RBAC Architecture](https://docs.feast.dev/v/master/getting-started/architecture/rbac) 
+- [RBAC Permission](https://docs.feast.dev/v/master/getting-started/concepts/permission).
+- [RBAC Authorization Manager](https://docs.feast.dev/v/master/getting-started/components/authz_manager)
+
+## Tools and Projects
+- Kubernetes
+- Feast
+- PostgreSQL Database
+- Keycloak(if oidc)
+
+## Application Environment
+
+This demo contains the following components:
+
+1. Feast Remote Server components (online, offline, registry).
+2. Feast Remote Client RBAC example.
+3. Yaml Configuration and installation related scripts files.
+ 
+![demo.jpg](demo.jpg)
+
+## Setup Instructions
+
+The application works under OpenShift or Kubernetes with the following instructions assume that you are using a OpenShift or Kubernetes cluster.
+
+### Prerequisites
+
+1. Kubernetes Cluster and Kubernetes CLI (kubectl).
+2. Helm: Ensure you have Helm installed for deploying the Feast components. 
+3. Python environment.
+4. Feast CLI latest version.
+
+## 1. Prerequisites Step
+
+ - **Step 1 : Create the Feast project with PostgreSQL.**
+
+    * Install the PostgreSQL on a Kubernetes cluster if you are using OpenShift you can install using [OpenShift Template](https://github.com/RHEcosystemAppEng/feast-workshop-team-share/tree/main/feast_postgres#1-install-postgresql-on-openshift-using-openshift-template)
+    * Port Forward PostgreSQL Database to your local machine. Since we are setting up the Feast project locally using the Feast CLI, so for local connection required port forward PostgreSQL:
```

Comment:
```suggestion
    * Port Forward the PostgreSQL Database to your local machine. Since we are setting up the Feast project locally using the Feast CLI, we need to port forward PostgreSQL:
```